src/middlewared/middlewared/plugins/pool.py

```diff
@@ -4182,6 +4182,147 @@ def __handle_zfs_set_property_error(self, e, properties_definitions):
             raise CallError(pjob.error)
         return data
 
+    @accepts(
+        Str('id', required=True),
+        Str('type', enum=['USER', 'GROUP']),
```

Comment:
Also perhaps we could rename it to something diffferent then just `type` as `type` variable is predefined in python

Reply:
I dont see a problem naming it type, is there an issue with it?

Reply:
Technically no, just a suggestion :p

Reply:
I think we should just rename the python variable to type_ and keep "type" for the API name

Reply:
Agreed :)

Reply:
Changed to quota_type